aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -1011,12 +1011,12 @@ module aptos_framework::fungible_asset {
 
     public(friend) fun deposit_internal(store_addr: address, fa: FungibleAsset) acquires FungibleStore, ConcurrentFungibleBalance {
         let FungibleAsset { metadata, amount } = fa;
-        if (amount == 0) return;
-
         assert!(exists<FungibleStore>(store_addr), error::not_found(EFUNGIBLE_STORE_EXISTENCE));
```

Comment:
Can you move this to the first line? Line 1013 is only needed for the subsequent metadata check

Reply:
This is nit 5 fix...